### ISSUE TITLE
Integrate random email for new entry

### ISSUE THREAD

**Issue by mahe8696** [Kunzisoft/KeePassDX#1860]
Both simple login and Addy are open source and bktwarden does this.

Since bktwarden is open source, can we not copy their code to integrate simple login and Addy API's

**Comment by shuvashish76** [comment]
I'm guessing your FR is about username i.e email aliases creation using their API?
No such features are not possible as KPDX has no internet permission in the 1st place.

**Comment by J-Jamet** [comment]
I agree, the principle is not to have Internet permission and not to integrate service APIs with non-generic protocols. Here it would be good to think of a dedicated application that would plug into KeePassDX to generate random emails for new credentials to register.
I'll think about it.

**Comment by mahe8696** [comment]
Yes Sir

**Comment by mahe8696** [comment]
Yes sir
